go/staking/api/slashing.go

```diff
@@ -32,10 +32,13 @@ const (
 	// SlashConsensusEquivocationName is the string representation of SlashConsensusEquivocation.
 	SlashConsensusEquivocationName = "consensus-equivocation"
 	// SlashBeaconInvalidCommitName is the string representation of SlashBeaconInvalidCommit.
+	// Deprecated: This was for the PVSS beacon.
```

Comment:
Probably we should update fixgenesis to remove these? And then we can also remove them here.

Reply:
Done.